x-pack/plugins/apm/server/routes/fleet.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createApmServerRoute } from './create_apm_server_route';
+import { createApmServerRouteRepository } from './create_apm_server_route_repository';
+
+const hasFleetDataRoute = createApmServerRoute({
+  endpoint: 'GET /api/apm/fleet/hasData',
```

Comment:
Endpoints should be snake case

```suggestion
  endpoint: 'GET /api/apm/fleet/has_data',
```